src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java

```diff
@@ -6998,8 +7036,43 @@ public void testListBusinessStakeholders_withUserTokenAndSomeOptions() throws Ex
         assertThat(resp, is(equalTo(response)));
 
         Mockito.verify(mockApiClient).get(Mockito.eq("https://api.sandbox.hyperwallet.com/rest/v4/users/" + token
-            + "/business-stakeholders?createdBefore=2016-06-29T17:58:26Z&limit=10"), Mockito.any(TypeReference.class));
+                + "/business-stakeholders?createdBefore=2016-06-29T17:58:26Z&limit=10"), Mockito.any(TypeReference.class));
+
+    }
 
+    @Test
+    public void listBusinessStakeholders_WithVerificationDocumentAndRejectReasons() throws Exception {
```

Comment:
minor: letter after underscore should be lower case to comply with other test method names